gbpservice/tests/contrib/nfp_service/reference_configurator/api/root_controller.py

```diff
@@ -11,8 +11,8 @@
 #    under the License.
 
 import pecan
-from gbpservice.tests.contrib.nfp_service.reference_configurator.\
```

Comment:
Checked and they are fine.

Regards,
Deepak

On Fri, Jun 17, 2016 at 11:44 AM, njagadish <notifications@github.com>
wrote:

> In
> gbpservice/tests/contrib/nfp_service/reference_configurator/api/root_controller.py
> <https://github.com/oneconvergence/group-based-policy/pull/445#discussion_r67464521>
> :
>
> > @@ -11,8 +11,8 @@
> >  #    under the License.
> >
> >  import pecan
> > -from gbpservice.tests.contrib.nfp_service.reference_configurator.\
>
> Please check PEP8 errors on these changes.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/oneconvergence/group-based-policy/pull/445/files/45bedf36aca03235516e3a571271860b16c9c258#r67464521>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AGq88wwa10wdEfzdsPfQ-8EEp2jEMEc4ks5qMjtZgaJpZM4I4DqM>
> .
>